content/docs/user-guide/components/reference/physx/prismatic-joint.md

```diff
@@ -0,0 +1,65 @@
+---
+linkTitle: PhysX Prismatic Joint
+description: ' Lean to use the Open 3D Engine PhysX Prismatic Joint component. '
+title: PhysX Prismatic Joint Component
+---
+
+
+
+With the **PhysX Prismatic Joint** component, you can create a dynamic prismatic joint that constrains an entity to the joint keeping the same rotation but allowing to move freely along one axis.
+
+## PhysX Prismatic Joint component properties 
+
+![Properties of the PhysX Prismatic Joint component](/images/user-guide/physx/physx/ui-physx-prismatic-joint-component.png)
+
+**Local Position**
+Specify the position of the joint relative to the entity transform.
+
+**Local Rotation**
+Specify the rotation of the joint relative to the entity transform.
+
+**Lead Entity**
+Specify the parent entity that will drive the joint.
```

Comment:
nit: Maybe say `...the lead (parent) entity...`

Reply:
It's the same sentence as in all the other joint types pages. I can change it in all of them.